yew/src/html/scope.rs

```diff
@@ -162,7 +162,11 @@ impl<COMP: Component> Scope<COMP> {
         scheduler().push_comp(ComponentRunnableType::Destroy, Box::new(destroy));
     }
 
-    /// Send a message to the component
+    /// Send a message to the component.
+    ///
+    /// Please be aware that currently this method
+    /// immediately/synchronously schedules a call to the
+    /// [Component](Component) interface. 
```

Comment:
chatting with @siku2 in discord, decided it might be a nice idea to add a reference to the `yewtil::LinkFuture::send_future()` which may be one solution to problems that users may encounter with using this method. Currently this would be blocked by https://github.com/yewstack/yew/issues/1233 because `LinkFuture` isn't released yet.